src/v2/Apps/Artwork/Components/ArtworkSidebar/CreateArtworkAlertSection.tsx

```diff
@@ -23,11 +47,83 @@ export const CreateArtworkAlertSection: React.FC = () => {
         <Text variant="xs" mr={2}>
           Be notified when a similar piece is available
         </Text>
-        <Button variant="secondaryOutline" size="small">
+        <Button variant="secondaryOutline" size="small" onClick={openModal}>
           <BellIcon mr={0.5} fill="currentColor" />
           <Text variant="xs">Create Alert</Text>
         </Button>
       </Flex>
+
+      {visible ? (
+        <SavedSearchAlertModal
+          initialValues={{ name: "", email: true, push: false }}
+          entity={entity}
+          onClose={closeModal}
+          onComplete={closeModal}
+        />
+      ) : null}
     </Box>
   )
 }
+
+export const CreateArtworkAlertSectionContainer: React.FC<CreateArtworkAlertSectionProps> = ({
+  artwork,
+}) => {
+  const artists = compact(artwork.artists)
+  const artistIDs = artists.map(artist => artist.internalID)
+  const placeholder = `Artworks like: ${artwork.title!}`
+  const attributionClass = getAttributionClassIdByLabel(
+    artwork.attributionClass?.name ?? ""
+  )
+
+  const entity: SavedSearchEntity = {
+    type: "artist", // TODO: Use OwnerType
```

Comment:
**suggestion:** Like you've written in the TODO this could be 
```suggestion
    type: OwnerType.artist, 
```

Reply:
Yes, that's right. I thought to do it in a separate PR, since there are already so many changes to check